cypress/support/step-definitions/batch-selection-steps.js

```diff
@@ -0,0 +1,61 @@
+import {
+  batchSelectionComponent,
+  batchSelectionCounter,
+  batchSelectionButtons,
+} from '../../locators/batch-selection';
+
+const FIRST_BUTTON = 0;
+const SECOND_BUTTON = 1;
+const THIRD_BUTTON = 2;
+
+Then('Batch selection component is rendered properly', () => {
```

Comment:
Try to reduce lines of code, I mean try to split this step for two separate 
1. Then('Batch selection component is rendered properly', )
2. And('Batch selection {string} button is rendered properly') - here try to use `for` to choose FIRST, SECOND, THIRD element. Use `Scenario Outline` to pass what value `data-element` should have

Reply:
divided onto two steps